hbase-server/src/test/java/org/apache/hadoop/hbase/master/balancer/TestStochasticBalancerJmxMetrics.java

```diff
@@ -199,11 +202,13 @@ public void testJmxMetrics_PerTableMode() throws Exception {
     }
   }
 
-  private Set<String> readJmxMetricsWithRetry() throws IOException {
+  private Set<String> readJmxMetricsWithRetry(int expectedMetricsCount) throws IOException {
     final int count = 0;
-    for (int i = 0; i < 10; i++) {
+    for (int i = 0; i < 20; i++) {
```

Comment:
just realized in another PR the count used on line 214 is never updated... TODO to fix.